tests/string_test.py

```diff
@@ -1,9 +1,7 @@
-#!/usr/bin/env python3
-
```

Comment:
string_tests also sporadically fails for me (~1/10 times):

```
=================================================== short test summary info ====================================================
FAILED tests/string_test.py::StringTest::test_index - assert '\\"\\\\@PSTZ 4' == '"\\@PSTZ 4'
================================================= 1 failed, 8 passed in 2.69s ==================================================
```

Reply:
Yeah, noticed that also and I believe that's an unrelated bug that I am gonna research.

Reply:
Is there a way to ignore failures until we can figure that out? I'm thinking about automated testing, where false-positives are a problem.

Reply:
Yup, can actually set ignore test for now, will add that.

Reply:
@ronawho I submitted a [ticket](https://github.com/mhmerrill/arkouda/issues/364) for the periodic string_test fail and added a unittest ignore annotation for now.